person.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'nameable'
+
+class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
```

Comment:
- [ ] I suggest that you make `rentals` an `attr_accessor` to be able to read and modify it